src/lucky/routeable.cr

```diff
@@ -138,6 +197,30 @@ module Lucky::Routeable
     \{% end %}
   end
 
+  # Access a query parameter
+  #
+  # This will allow any action to accept and access query parameters. By
+  # default query parameters are ignored, but adding a `param` will make them
```

Comment:
I'm not sure I understand the "by default a query paramter will be ignored. It might be more clear to omit that phrase and instead say something like: `It will set up a method that can be used to access the parameter. If the type of the param does not allow `Nil`, then the query parameter will be required when generating paths and urls.`

Reply:
I think I was trying to say that if you don't use `param` any query parameter in the url is not in some params object and therefore inaccessible. The "setup a method" bit still feels a little but too much like describing the implementation not the effect to me, but I agree with your sentiment. I'll find another way.

Reply:
Ah, I see. This isn't _quite_ right though. You can access params with `params.get(:page)`. It just isn't as safe, because it may or may not be there

Reply:
Ah gotcha. Ok, I'll incorporate that also.